Bottom: 0c645cbe633603e94823152262061516f9d7da71
Top:    35df34c0d1839221f8efd58936e3a13861bea590
Author: Ian Kent <ikent@redhat.com>
Date:   2013-04-04 14:41:07 +0800

Refresh of autofs-5.0.7-syncronize-handle_mounts-shutdown.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 9e83913..ec22fa0 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -85,9 +85,6 @@ pthread_attr_t th_attr_detached;
 struct master_readmap_cond mrc = {
 	PTHREAD_MUTEX_INITIALIZER, PTHREAD_COND_INITIALIZER, 0, NULL, 0, 0, 0, 0};
 
-struct startup_cond suc = {
-	PTHREAD_MUTEX_INITIALIZER, PTHREAD_COND_INITIALIZER, 0, 0};
-
 struct shutdown_cond fc = {
 	PTHREAD_MUTEX_INITIALIZER, PTHREAD_COND_INITIALIZER, 0, 0, 0};
